build.gradle

```diff
@@ -1,34 +1,20 @@
+plugins {
+  id 'base'
+  id "io.spinnaker.project" version "$spinnakerGradleVersion" apply false
```

Comment:
grabbing this as a dependency gives us the BintrayPublishPlugin and transitively the ospackage plugin, but we don't want to apply the io.spinnaker.project because this isn't a multiproject jvm app + application-packaged-as-ospackage

Reply:
Why don't we just depend on those plugins directly? I guess maybe the advantage of this is it means we use the same versions of bintray and ospackage as the rest of the projects?

Reply:
Oh, I didn't realize `BintrayPublishPlugin` was _our_ plugin... I thought it was one from the main gradle plugins repo. Thanks makes sense...

(Realized from your comment in the monitoring PR)